datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -122,6 +122,11 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @ApiOperation(value = "Re-size SDX cluster", produces = "application/json", nickname = "resizeSdx")
     SdxClusterResponse resize(@PathParam("name") String name, @Valid SdxClusterResizeRequest resizeSdxClusterRequest);
 
+    @POST
+    @Path("{name}/refresh")
+    @Produces(MediaType.APPLICATION_JSON)
+    SdxClusterResponse refresh(@PathParam("name") String name, @QueryParam("datahubName") String datahubName);
```

Comment:
We are not refreshing datalake. We should make it clear here.

Can you change it to refreshDataHubs?
Change the name to datalakeName?